config/inference_router_config.json

```
@@ -0,0 +1,45 @@
+{
+  "basic": {},
+  "health_check": {
+    "check_frequency": "10s",
+    "consecutive_success": 2,
+    "consecutive_failure": 2
+  },
+  "servers": [
+    {
+      "name": "main-server",
+      "addr": "0.0.0.0:8443,::1:8443",
```

In order to listen to external requests on 8443 for both ipv4 and ipv6 we need to use:

```suggestion
      "[::]:8443",
```

---

Done.